lib/adyen/rest/client.rb

```diff
@@ -19,15 +20,14 @@ module REST
     class Client
       include AuthorisePayment
       include ModifyPayment
+      include Payout
 
       attr_reader :environment
 
       # @param environment [String] The Adyen environment to interface with. Either
       #   <tt>'live'</tt> or <tt>'test'</tt>.
-      # @param username [String] The webservice username, e.g. <tt>ws@Company.Account</tt>
-      # @param password [String] The password associated with the username
-      def initialize(environment, username, password)
-        @environment, @username, @password = environment, username, password
+      def initialize(environment)
+        @environment = environment
```

Comment:
I wanted to avoid something like that, but yeah I'm not a big fan either of calling   `Adyen.configuration` directly in Request :/ At the end I guess it's better to stick with what we already have.

May I ask why the deprecated_username/password?